### PR TITLE
Feat: Added bias correction option to PTQ algorithms

### PR DESCRIPTION
**Pull request by nickfraser**
Added option to apply the bias correction PTQ algorithm to the example. I also fixed some minor typos in the documentation.

### CODE REVIEW

File: optimum/amd/brevitas/configuration.py [hunk 1]
```diff
@@ -54,7 +54,9 @@ class BrevitasQuantizationConfig:
...
             - `"cross_layer"`: Apply SmoothQuant, and fuse the activation rescaling within a preceding layer when possible (example: nn.LayerNorm followed by nn.Linear). This is achieved through a graph capture of the model using [torch.fx](https://pytorch.org/docs/stable/fx.html#module-torch.fx).
         apply_weight_equalization (`bool`, defaults to `False`):
-            Applies weight equalization accross layers, following https://arxiv.org/abs/1906.04721. This parameter is useful for models whose activation function is linear or piecewise-linear (like ReLU, used in OPT model), and allows to reduce the quantization error of the weights by balancing scales accross layers.
+            Applies weight equalization across layers, following https://arxiv.org/abs/1906.04721. This parameter is useful for models whose activation function is linear or piecewise-linear (like ReLU, used in OPT model), and allows to reduce the quantization error of the weights by balancing scales across layers.
+        apply_bias_correction (`bool`, defaults to `False`):
+            Applies bias correction to compensate for changes in activation bias caused by quantization
```
Comment (fxmarty) [inline optimum/amd/brevitas/configuration.py:59]:
```suggestion
            Applies bias correction to compensate for changes in activation bias caused by quantization.
```

Can you also link to a reference?

File: optimum/amd/brevitas/quantizer.py [hunk 2]
```diff
@@ -206,6 +206,14 @@ def quantize(
             apply_calibration(model, calibration_dataset)
             logger.info("Activation calibration applied.")
 
+        if quantization_config.apply_gptq:
```
Comment (fxmarty) [inline optimum/amd/brevitas/quantizer.py:209]:
```suggestion
        if quantization_config.apply_bias_correction:
```

File: optimum/amd/brevitas/configuration.py (hunk 1 above)
Comment (nickfraser) [reply optimum/amd/brevitas/configuration.py:59]: The implementation is loosely based on the concepts in the weight equalisation [paper](https://arxiv.org/abs/1906.04721), but we use calibration data to generate the statistics, instead of relying on batchnorm + other linear transforms to estimate them. Not sure we have a specific reference for that, but I think other works are using a similar approach...
